pkg/server/api/agent/v1/service.go

```diff
@@ -397,8 +395,8 @@ func (s *Service) RenewAgent(ctx context.Context, req *agentv1.RenewAgentRequest
 	}
 
 	// Agent attempted to renew when it should've been reattesting
-	if attestedNode.CanReattest && fflag.IsSet(fflag.FlagReattestToRenew) {
-		return nil, errorutil.PermissionDenied(types.PermissionDeniedDetails_AGENT_MUST_REATTEST, "agent can't renew SVID, must reattest")
+	if attestedNode.CanReattest {
+		log.Warn("Agent renewing SVID when it should be reattesting")
```

Comment:
1.8.x agents calling this API will always trigger this warning unless they have the feature flag enabled.
It doesn't seem that we can safely enforce reattestation until 1.10 without breaking existing agents.
We should probably just add a TODO comment here to change this to return an error in v1.10.